### PR TITLE
Add FreeMarker icon

### PR DESCRIPTION
**Pull request by dcortes92**
**Changes proposed:**
* [x] Add
* [ ] Prepare
* [ ] Delete
* [ ] Fix

**Things I've done:**
* [ ] My pull request fixes an issue, I referenced the issue.

Adding icon for [FreeMarker](http://freemarker.org/) files.

### CODE REVIEW

**Comment by robertohuertasm** [comment]
Agreed. Below is the favicon `FreeMarker` is using. We should have it if possible.

image

**Comment by JimiC** [comment]
@dcortes92 If you are fine with the requested change, I can provide the icon later in the day.

**Comment by dcortes92** [comment]
Hi @JimiC, sure I didn't have an icon for this one so I used the same as in other editors. Let me know when you're able to provide the new icon.

Regards,

Daniel

File: src/icon-manifest/languages.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const languages: ILanguageCollection = {
   fsharp: { ids: 'fsharp', defaultExtension: 'fs' },
   diff: { ids: 'diff', defaultExtension: 'diff' },
   dockerfile: { ids: 'dockerfile', defaultExtension: 'dockerfile' },
+  ftl: { ids: 'ftl', defaultExtension: 'ftl' },
```
Comment (JimiC) [inline src/icon-manifest/languages.ts]: Please modify to: 
```
freemarker: { ids: ['ftl'], defaultExtension: 'ftl' },
```

File: src/icon-manifest/supportedExtensions.ts [hunk 2]
```diff
@@ -134,6 +134,7 @@ export const extensions: IFileCollection = {
     { icon: 'fortran', extensions: ['f90', 'mod', 'f'], format: FileFormat.svg },
     { icon: 'fsharp', extensions: [], languages: [languages.fsharp], format: FileFormat.svg },
     { icon: 'fsproj', extensions: ['fsproj'], format: FileFormat.svg },
+    { icon: 'ftl', extensions: [], languages: [languages.ftl], format: FileFormat.svg },
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: Please modify to:
```
{ icon: 'freemarker', extensions: [], languages: [languages.freemarker], format: FileFormat.svg },
```

File: src/models/language/languageCollection.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ export interface ILanguageCollection {
   fsharp: ILanguage;
   diff: ILanguage;
   dockerfile: ILanguage;
+  ftl: ILanguage;
```
Comment (JimiC) [inline src/models/language/languageCollection.ts]: Please modify to:
```
freemarker: ILanguage;
```

**Comment by JimiC** [comment]
FreeMarker icon: https://github.com/JimiC/vscode-icons-svg/blob/master/icons/file_type_freemarker.svg

**Pull request merged**